### PR TITLE
Feat/profile pages tooltips

### PR DESCRIPTION
**Pull request by tsubik**
This PR introduces tooltips to both profile actor and place pages (https://github.com/Vizzuality/trase/issues/248).

Tooltips were added to the places described in the given spreadsheet.

One spec (from version_migrations) has been removed as discussed with @agnessa it was no longer relevant.

Travis build was slightly broken so I fixed it here too.

### CODE REVIEW

**Comment by tiagojsag** [comment]
Hi

Just a couple of minor details:
- [ ] On the actor header section, the tooltip icons seem to have the correct size but need to be moved down a bit
- [ ] On the remaining places, the tooltip icons seem to be smaller than on the design

https://app.zeplin.io/project/5a017b22d882ec6e04009065/screen/5a0ecf9ed767e60131c940c4

Otherwise looks really nice :+1:

**Comment by tsubik** [comment]
Ok. Done, but as I fixed travis then now I have not related tests failing after merge with develop branch cc @agnessa

**Comment by agnessa** [comment]
Please cherry pick: https://github.com/Vizzuality/trase/commit/62feadde0f8dc3b08e6a85795a4ae24338ca76c7

Not sure why this wasn't failing in develop before 😕

**Comment by tsubik** [comment]
@agnessa Tests on CI probably stopped running after moving frontend part into this repo, take a look here https://travis-ci.org/Vizzuality/trase/builds/344884552#L989. I fixed travis config file in that PR.

**Comment by tiagojsag** [comment]
That's fairly recent, as we only added FE tests a few weeks ago cc @sorodrigo

**Comment by sorodrigo** [comment]
It's weird that it needed a `cd ..` because as @agnessa mentions, it was passing before. Wouldn't it be cleaner to use a subshell? would it even work? `(cd frontend; npm test;)`

**Comment by tiagojsag** [comment]
AFAICT it passed because it tried to run rails tests where there were none, so no tests were failing... yeah, weird

**Comment by sorodrigo** [comment]
@tiagojsag fixed some edge cases where tooltips went out of the viewport. Please review

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -46,6 +47,12 @@
     Dictionary::Quant.instance.reset
     Dictionary::Qual.instance.reset
     Dictionary::Ind.instance.reset
+
```
Comment (tiagojsag) [inline spec/rails_helper.rb]: this feels out of place in this PR. @tsubik @agnessa ?

Comment (tsubik) [reply spec/rails_helper.rb]: @tiagojsag This is cherry-picked from @agnessa branch to stop unrelated tests erroring on CI after merging in develop branch.
